Makefile

```
@@ -38,6 +38,8 @@ else
 	REGISTRY = $(RP_IMAGE_ACR)
 endif
 
+ARO_PORTAL_BUILD_IMAGE ?= $(ARO_IMAGE_BASE):$(VERSION)
+ARO_BUILDER_IMAGE ?= $(ARO_IMAGE_BASE):$(VERSION)
```

These images are currently the same, and are the same as ARO_IMAGE as well below. We likely need all 3 to be unique. I like the naming convention used in the env file, can we replicate those names here?

---

I'd suggest explicitly pointing these to localhost images (similar to what you did in env.example below):

```suggestion
ARO_PORTAL_BUILD_IMAGE ?= localhost/aro-portal-build:$(VERSION)
ARO_BUILDER_IMAGE ?= localhost/aro-builder:$(VERSION)
```

---

Done. Not added version for building local images because --cache-from functionality neither uses tags nor it uses digests.